openedx/openedx-platform
# feat: export tagged course as csv

## Description
This PR implements a feature to export a CSV with the course structure and its blocks, showing all tags applied to them.

## More information
Part of:
- https://github.com/openedx/modular-learning/issues/172

Depends on:
- https://github.com/openedx/edx-platform/pull/34143

## Testing instructions
1. Check the tests cover the implemented feature
1. Follow the instructions from https://github.com/openedx/frontend-app-course-authoring/pull/830 to test it from the MFE

___
Private-ref: [FAL-3610](https://tasks.opencraft.com/browse/FAL-3610)
